src/platform/packages/shared/content-management/table_list_view_table/src/table_list_view.test.tsx

```diff
@@ -330,6 +331,178 @@ describe('TableListView', () => {
     });
   });
 
+  describe('column sorting with custom columns', () => {
+    const customSortingOptions: CustomSortingOptions = {
+      field: 'typeTitle',
+      sortingLabels: [
+        {
+          label: 'Type A-Z',
+          direction: 'asc',
+        },
+        {
+          label: 'Type Z-A',
+          direction: 'desc',
+        },
+      ],
+    };
+    const customTableColumn = {
+      field: 'typeTitle',
+      name: 'Type',
+      sortable: true,
+    };
+
+    const setupCustomColumnSorting = registerTestBed<string, TableListViewTableProps>(
+      WithServices<TableListViewTableProps>(TableListViewTable, {
+        TagList: getTagList({ references: [] }),
+      }),
+      {
+        defaultProps: { ...requiredProps, customSortingOptions, customTableColumn },
+      }
+    );
+
+    const hits: Array<UserContentCommonSchema & { typeTitle: string }> = [
+      {
+        id: '123',
+        updatedAt: twoDaysAgo.toISOString(),
+        type: 'maps',
+        attributes: {
+          title: 'Item 2',
+        },
+        references: [],
+        typeTitle: 'Vega',
+      },
+      {
+        id: '456',
+        updatedAt: yesterday.toISOString(),
+        type: 'vega',
+        attributes: {
+          title: 'Item 1',
+        },
+        references: [],
+        typeTitle: 'Lens',
+      },
+    ];
+
+    test('should display 6 sorting options when custom columns are present', async () => {
```

Comment:
if I am not mistaken, this test is identical to the start of the next test? Then I think we should just remove it because everything it does is checked inside the 2nd test